rust/auction/program/src/processor/place_bid.rs

```diff
@@ -235,7 +235,7 @@ pub fn place_bid<'r, 'b: 'r>(
             args.amount,
             min[0]
         );
-        if args.amount <= min[0] {
+        if args.amount < min[0] {
```

Comment:
🤔

Reply:
If the minimum is 1 and you bid 1, you can't bid it. With this fix, you can now bid 1. Nothing special here. :)